piwik-sdk/src/main/java/org/piwik/sdk/Piwik.java

```diff
@@ -68,6 +68,16 @@ public synchronized Tracker newTracker(@NonNull String trackerUrl, int siteId) t
     }
 
     /**
+     * Configure Piwik to receive more than 5 custom variable
+     * http://piwik.org/faq/how-to/faq_17931/
+     * Changing this limit will affect all Tracker instances
+     * @param maxVariables
+     */
+    public void setMaxCustomVariables(int maxVariables){
+        CustomVariables.MAX_VARIABLES = maxVariables;
```

Comment:
@d4rken I tried to avoid static vars, but my shot wasn't successful and elegant due to [this usage of CustomVariables](https://github.com/piwik/piwik-sdk-android/blob/8e1c23d45e37f7976db5d12f72b360a313001177/piwik-sdk/src/main/java/org/piwik/sdk/TrackMe.java#L23) in `TrackMe` class, it knows nothing about `Tracker` instance.